examples/README.md

```diff
@@ -14,15 +14,44 @@ and streaming APIs.
 You can find examples that demonstrate these APIs
 in the [`examples/src/`](./src) directory.
 
+* [How to use the one-step APIs](./src/onestep_defaults.py)
+* [How to change the algorithm suite](./src/onestep_unsigned.py)
```

Comment:
```suggestion
* [How to really screw up your security model for fun and profit](./src/onestep_unsigned.py)
```

Seriously, do we want to show this? We don't recommend it.

Reply:
The example I show is using the AES-GCM-256-HKDF-SHA256 algorithm suite. The default but without the full message signature.

In the example description I go into more details about why you might want to turn off the full message signature and how to tell if it will change your security model.